packages/curve-ui-kit/package.json

```diff
@@ -17,10 +17,27 @@
     "@mui/icons-material": "^6.1.4",
     "@mui/material": "^6.1.4",
     "@mui/utils": "^6.1.4",
+    "@safe-global/safe-apps-provider": "^0.18.3",
+    "@safe-global/safe-apps-sdk": "^9.1.0",
     "@tanstack/query-sync-storage-persister": "^5.59.0",
     "@tanstack/react-query": "^5.59.0",
     "@tanstack/react-query-devtools": "^5.59.0",
     "@tanstack/react-query-persist-client": "^5.59.0",
+    "@web3-onboard/bitget": "^2.1.1",
```

Comment:
:exploding_head:

Reply:
If one day we migrate from web3-onboard to wagmi we'll be able to get rid of all of them I think.